pages/_app.js

```diff
@@ -1,10 +1,30 @@
 import GlobalStyle from "../styles";
+import { initialCountries } from "@/lib/db";
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { uid } from "uid";
 
 export default function App({ Component, pageProps }) {
+  const [isCountry, setCountry] = useState(initialCountries);
+  const router = useRouter();
+  function submitNewCountry(country) {
+    const newCountries = {
```

Comment:
```suggestion
    const newCountry = {
```
Are you submitting one country or multiple? As far as I can tell, you're only submitting one. 🤔